.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 12.x
-      - run: yarn install --frozen-lockfile --ignore-engines
+      - run: yarn install --ignore-engines
```

Comment:
Why is this removed? Doesn't seem to be in the scope of this PR

Reply:
Not only here, but also in the rest of the file

Reply:
Done. Actually, --frozen-lockfile was giving an error as it wasn't allowing dependent packages to be installed i.e. ibm_db in my case and was erroring. Not sure if I missed something but I will give it another try.

Reply:
I retried and error I am getting is as follows:
Run yarn install --frozen-lockfile --ignore-engines
yarn install v1.22.17
[1/4] Resolving packages...
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

**Can you please suggest how to rectify this?**

Reply:
Locally you can run `yarn install --ignore-engines` the first time and then you commit the new yarn.lock file. Then it should work fine afterwards.